pkg/apis/policy/validation/validation_test.go

```diff
@@ -906,3 +906,30 @@ func TestValidatePSPRunAsGroup(t *testing.T) {
 		})
 	}
 }
+
+func TestValidatePSPSELinux(t *testing.T) {
+	var testCases = []struct {
+		name    string
+		selinux policy.SELinuxStrategyOptions
+		fail    bool
+	}{
+		{"SELinuxStrategyMustRunAs",
```

Comment:
nit: put the opening and closing braces on separate lines, to make the indentation more consistent.